app/stores/toplevel/ProfileStore.js

```diff
@@ -326,6 +333,26 @@ export default class ProfileStore extends Store {
     );
   }
 
+  // ========== Show/hide Balance ========== //
+
+  @computed get shouldHideBalance(): boolean {
+    const { result } = this.getHideBalanceRequest.execute();
+    return result === true;
+  }
+
+  @computed get isHideBalanceSet(): boolean {
```

Comment:
yeah the functions that check if a `Request` was executed is usually unused for Request calls that modify LocalStorage. The whole way we setup these calls for modifying LocalStorage is messy in general because it was written this way to look stylistically similar to the rest of the codebase despite being for synchronous calls. I try not to think about it and just keep coding 🙈

Reply:
That's it, I was following the patterns of the codebase. I'll remove that function ;)